_posts/2021-05-17-changelog-147.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #147"
+date: 2021-05-17 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#7140] Enable [new name resolution engine](https://blog.jetbrains.com/clion/2020/12/intellij-rust-updates-for-2020-3/#new-resolve) by default
+
+* [#6456] Add WSL support. To use it, enable `org.rust.wsl` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#7102], [#7165] Annotate unknown cfg attributes and provide a quick-fix. This corresponds to the [E0537](https://doc.rust-lang.org/error-index.html#E0537) compiler error. (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-147/unknown_cfg_predicate" w="700px" %}
+
+* [#6832] Suggest target triples in command for Cargo run configuration and run anything dialog. Suggest installing the target if it is not yet installed
+
+    <img src="/assets/posts/changelog-147/suggest_target_triple.png" alt="Target triple completion in cargo run configuration" width="700px"/>
+
+* [#7098] Support basic resolve of macros 2.0 (`pub macro name() { ... }`)
+
+## Performance Improvements
+
+* [#7181] Slightly optimize cargo refresh when using new name resolution engine
```

Comment:
"cargo refresh" is a more internal term. Let's rephrase it somehow. You can use "project model update" term or something similat